index.d.ts

```diff
@@ -55,6 +55,7 @@ export interface Config {
     live_preview?: LivePreview;
     plugins?: ContentstackPlugin[];
     fetchOptions?: FetchOptions;
+    ea_headers?: string[]
```

Comment:
Is it public or private, Is user able to see this variable ? if that is the case then make this variable early_access?

Reply:
- Could you please rename the variable "header-ea" to "early_access"? It's important to use a more explicit name as users may not readily understand the abbreviation for "early access." We also don't need to disclose the internal handling of user input like how we are handling it.

- I'm not able to locate the tests based on early access implementation.

Rest look good to me @nadeem-cs

Reply:
Done with the changes. @ishaileshmishra